.eslintrc

```diff
@@ -1,14 +1,16 @@
 {
   "root": true,
   "extends": "airbnb-base",
+  "env": {
+    "jest": true,
+  },
```

Comment:
This env should be just in the test overrides I think, not at the top level.

Reply:
oh got it